pkgs/applications/science/logic/abc/libabc.nix

```diff
@@ -0,0 +1,41 @@
+{ fetchhg, stdenv, pkgs }:
+
+stdenv.mkDerivation rec {
+
+  # Note that this derivation provides only the library, but under the
+  # "abc" name, to associate with libabc, which conforms to the
+  # standard library dependency referencing nomenclature.  The binary
+  # is available as a separate package with the "abc-verifier" naming.
+
+  name = "abc-${version}";
+  version = "20160818";
+
+  src = fetchhg {
+    url    = "https://bitbucket.org/alanmi/abc";
+    rev    = "a2e5bc66a68a72ccd267949e5c9973dd18f8932a";
+    sha256 = "09yvhj53af91nc54gmy7cbp7yljfcyj68a87494r5xvdfnsj11gy";
+  };
+
+  buildInputs = [ ];
```

Comment:
Leftover line?

Reply:
Actually, I was being explicit, but I can remove that and update this pull request shortly.